packages/next/src/build/templates/pages.ts

```diff
@@ -1,21 +1,14 @@
-// @ts-ignore this need to be imported from next/dist to be external
```

Comment:
is this not valid anymore?

Reply:
Because the import hits the ".js" `.module.compiled` file which does import via `next/dist`, it seems like this was able to allow them to remain external while enabling us to use the relative import 😌